README.md

```diff
@@ -89,6 +89,34 @@ ReactDom.render(
 );
 ```
 
+## Troubleshooting
```

Comment:
If I may suggest, instead writing it to readme file, create new Troubleshooting.md and all this issues/problems can land there.
I can add my experience with custom endpoints with data providers and generator.